scipy/spatial/transform/_rotation.pyx

```diff
@@ -223,6 +236,140 @@ cdef double[:, :] _compute_euler_from_matrix(
 
     return angles
 
+@cython.boundscheck(False)
+@cython.wraparound(False)
+cdef double[:, :] _compute_euler_from_quat(
```

Comment:
I've checked that addining `@cython.cdivision(True)` might be slightly beneficial for performance.

It will eliminate additional checks for the division here `cos_theta_2 = 2*(a**2 + b**2)/norm_squared - 1`

Reply:
I'm not at all an expert on Cython, so I'll believe you here!

Reply:
The effect is removing Python checks for division errors. 

The way to see is to run `cython -a source.pyx` and examine the generated html like (yellow lines will indicated overheard calls).

Reply:
Now as you've removed the division, this directive is not necessary.